exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,37 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+function capitalize(name) {
+    return name.toUpperCase();
+}
+// var result = capitalize(mentor1);
+// console.log(result);
+var shoutyMessage = "hello"
+
+function greeting(string){
+ return string.toUpperCase();
+ 
+}
+// var result2 = greeting(shoutyMessage);
+
+// console.log(result2);
+
+function shoutyGreeting(name, string){
```

Comment:
Well done Nishka 💯 
Maybe consider moving shoutyMessage variable into the shoutyGreeting function , since its only used in the function 
```suggestion
function shoutyGreeting(name, string){
  let shoutyMessage = "hello"
  ... rest of function

```